### PR TITLE
New available GCP region: northamerica-northeast2

### PR DESCRIPTION
**Pull request by than-pet**
## What is the purpose of the change

We are releasing a new datacenter in Canada region as described here
https://github.com/camunda/product-hub/issues/995

## When should this change go live?

@felix-mueller can answer this
SRE part is done and can be released any time soon (https://github.com/camunda-cloud/team-sre/issues/399)

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: versioned_docs/version-8.1/reference/regions.md [hunk 1]
```diff
@@ -20,5 +20,6 @@ Running on a Trial and want to try a different region, or interested in other re
 - US East: us-east1
 - US Central: us-central1
 - Australia Southeast: australia-southeast1
+- North America Northeast: northamerica-northeast2
```
Comment (than-pet) [inline versioned_docs/version-8.1/reference/regions.md:23]: probably this has to be on 8.2. But I don't know the release process tbh

Comment (felix-mueller) [reply versioned_docs/version-8.1/reference/regions.md:23]: from my point of view we could release new regions immediately, let's not wait for minor release

**Review by felix-mueller — APPROVED** [review]

**Comment by akeller** [comment]
@than-pet, thanks for your first docs contribution! Can you please sign the CLA?

**Comment by akeller** [comment]
If we want this available in the docs immediately, we can do a docs release. Just let me know! 

Otherwise, we'll cut the minor release docs on Monday US afternoon.

**Comment by than-pet** [comment]
> If we want this available in the docs immediately, we can do a docs release. Just let me know!
> 
> Otherwise, we'll cut the minor release docs on Monday US afternoon.

it's not urgent, you can release whenever you want

**Comment by akeller** [comment]
> [CLA assistant check](https://cla-assistant.io/camunda/camunda-platform-docs?pullRequest=1910) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/camunda/camunda-platform-docs?pullRequest=1910) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/camunda/camunda-platform-docs?pullRequest=1910) it.

@than-pet visit the CLA bot in this PR. It will have a link to sign the CLA - contributor license agreement.

**Comment by akeller** [comment]
Thanks all! I'll enable auto-merge and release shortly. 😄

**Pull request merged**